Client/include/ChronoKeeperClient.h

```diff
@@ -0,0 +1,31 @@
+#ifndef CHRONOLOG_KEEPER_CLIENT_H
+#define CHRONOLOG_KEEPER_CLIENT_H
+
+#include <vector>
+#include <map>
+
+#include <ChronoKeeperIdCard.h>
+
+namespace chronolog
+{
+
+template <typename RPCChannel>
+class ChronoKeeperClient
+{
+public:
+	ChronoKeeperClient( std::vector<KeeperIdCard> const&);
+	~ChronoKeeperClient();
+
```

Comment:
You could use the existing ChronoLog Client APIs and not have a separate class. You can have KeeperConnect and KeeperDisconnect APIs in the Keeper process and add those functions to client.h, RPCClient and client.cpp

Reply:
the way I see it is that ChronoLog Client API will contain the existing VisorRPC client , this ChronoKeeperClient that's just a stub for hte time being, and  time adjustment functionality, etc. 
VisorRPC  client and ChronoKeeperClient would have to use different Thalium engines